### PR TITLE
Add FSBP controls

### PR DESCRIPTION
**Pull request by karanpopat**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: query/networkfirewall/networkfirewall_firewall_policy_default_stateless_action_check_fragmented_packets.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  arn as resource,
+  case
+    when (not (firewall_policy -> 'StatelessFragmentDefaultActions') ? 'aws:drop'
+        and not (firewall_policy -> 'StatelessFragmentDefaultActions') ? 'aws:forward_to_sfe') then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when (not (firewall_policy -> 'StatelessFragmentDefaultActions') ? 'aws:drop'
+        and not (firewall_policy -> 'StatelessFragmentDefaultActions') ? 'aws:forward_to_sfe') then title || ' stateless action is neither drop nor forward for fragmented packets'
+    else title || ' stateless action is either drop or forward for fragmented packets'
```
Comment (misraved) [inline query/networkfirewall/networkfirewall_firewall_policy_default_stateless_action_check_fragmented_packets.sql:12]:
```suggestion
        and not (firewall_policy -> 'StatelessFragmentDefaultActions') ? 'aws:forward_to_sfe') then title || ' stateless action is neither drop nor forward for fragmented packets.'
    else title || ' stateless action is either drop or forward for fragmented packets.'
```

File: query/sns/sns_topic_notification_delivery_status_enabled.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      and sqs_failure_feedback_role_arn is null then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when application_failure_feedback_role_arn is null 
+      and firehose_failure_feedback_role_arn is null 
+      and http_failure_feedback_role_arn is null 
+      and lambda_failure_feedback_role_arn is null 
+      and sqs_failure_feedback_role_arn is null then title || ' has delivery status logging for notification messages enabled.'
+    else title || ' has delivery status logging for notification messages disabled.'
```
Comment (misraved) [inline query/sns/sns_topic_notification_delivery_status_enabled.sql:18]: I think the status and reason columns have been reversed. For instance, the `alarm` status is going to have the reason that `title || ' has delivery status logging for notification messages enabled.`

File: query/sns/sns_topic_notification_delivery_status_enabled.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  end as reason,
+  -- Additional Dimensionswaf
```
Comment (misraved) [inline query/sns/sns_topic_notification_delivery_status_enabled.sql:20]:
```suggestion
  -- Additional Dimensions
```

File: foundational_security/docs/foundational_security_waf_6.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+**To encrypt an unencrypted SNS topic**
+
+1. Open the [Amazon WAF console](https://console.aws.amazon.com/wafv2/).
+2. In the navigation pane, choose `Rules`.
+3. Choose the name of the rule to update.
+4. Choose `Edit`.
+5. To add a condition to the rule, specify the following values:
+    
+  **When a request does/does not**
```
Comment (misraved) [inline foundational_security/docs/foundational_security_waf_6.md:19]:
```suggestion
  **When a request does/does not**
  
```

File: foundational_security/docs/foundational_security_waf_6.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    If you want AWS WAF Classic to allow or block requests based on the filters in a condition, choose does. For example, if an IP match condition includes the IP address range 192.0.2.0/24 and you want AWS WAF Classic to allow or block requests that come from those IP addresses, choose does.
+    If you want AWS WAF Classic to allow or block requests based on the inverse of the filters in a condition, choose does not. For example, if an IP match condition includes the IP address range 192.0.2.0/24 and you want AWS WAF Classic to allow or block requests that do not come from those IP addresses, choose does not.
+
+  **match/originate from**
```
Comment (misraved) [inline foundational_security/docs/foundational_security_waf_6.md:23]:
```suggestion
  **match/originate from**
  
```

**Review by rajlearner17 — APPROVED** [review]

**Pull request merged**